src/Symfony/Bundle/Resources/config/http_cache_purger.xml

```diff
@@ -24,5 +24,6 @@
             <argument type="service" id="api_platform.iri_converter" />
             <argument type="service" id="api_platform.http_cache.purger" on-invalid="null" />
         </service>
+
```

Comment:
can you remove this empty line we didn't touched that file

Reply:
Fixed in https://github.com/api-platform/core/pull/5758/commits/828ecb60ffeab01c34c9b6f922b208832d3a9d3d